index.html

```diff
@@ -836,17 +698,17 @@
       <ol>
         <li>
           Reading an <a>NFC tag</a> containing a <a>Web NFC message</a>, when a
-          web page using the Web NFC API is open and in <a>focus</a>.
+          web page using the Web NFC API is open and in <a data-cite="html#focus">focus</a>.
```

Comment:
what does the xref UI say about how to cite this?

Reply:
oh, we integrated the UI into ReSpec today (do a "shift-refresh")... if you click on the ReSpec pill, you can access the UI by clicking "Search Definitions".

Reply:
Oh, it doesn't show up... it means this is not exported! That means something might be wrong... either HTML needs to export it or you have a spec bug! 🚨

Reply:
Yes, spec bug! Two issues. You want this one instead:
https://html.spec.whatwg.org/multipage/interaction.html#focused

But it's not exported... you need to find out why? And if you think it should be exported from HTML, you need to ask the HTML folks to export it (or send them a PR, which adds `data-export=""` to the `dnf` in the HTML spec).

Reply:
OK, I will drop a PR to the HTML spec.

Reply:
Done, at https://github.com/whatwg/html/pull/4811